### DISCUSSION TITLE
Nuxt 3 and tree-shaking

### DISCUSSION THREAD

**Discussion by smares** [FortAwesome/Font-Awesome#19442] [Q&A]
Following the Docs on integrating FA with Nuxt 3 (https://fontawesome.com/docs/web/use-with/vue/use-with), importing a whole icon set results in a large bundle size. The alternative would be importing individual icons and adding them to the library.

Is there a way to import the whole set and have Vite tree-shake the bundle and remove unused icons?

**Comment by ghost** [comment]
I'm also really curious. Have you found something for this?
